### ISSUE TITLE
How to know which reads the kmer comes from

### ISSUE THREAD

**Issue by zhoudreames** [voichek/kmersGWAS#5]

**Comment by voichek** [comment]
To find the reads containing a _k_-mer of interest you have to do two things. First, you need to find in which accessions/strains the _k_-mer was found. This can be done using the **filter_kmers** functionality. Then you need to filter the reads and look for reads from the sample of interest and filter the ones that contain your _k_-mer. You can do this by writing your own script, I wrote a small program to do that but as I have seen there are [specialised program](https://github.com/7PintsOfCherryGarcia/RiDiCulousFilter) that do just that, I didn't find a reason to upload mine.

If you want I can upload my program as well. Let me know.

**Comment by zhoudreames** [comment]
Thank you very much for your help！If I can, I'd love to use your program，thank you again！

**Comment by voichek** [comment]
I will try to find time next week to organize the code and upload it.

**Comment by sunnycqcn** [comment]
Hello,
I still can not run filter_kmers functionality well.  I used command as following:
filter_kmers -t kmers_table -k LePR1/kmers/output/phenotype_value.assoc.txt -o ll1
I got the error is as following:
terminate called after throwing an instance of 'std::logic_error'
  what():  Ilegal kmer
Aborted (core dumped)
I think I should use a wrong associated kmer table. But I can not figure out this table. Coudl you suggest which table is assocated kmers table for parameter -k.
Thanks,
Fuyou
